ci/do_ci.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
 set -e
 
-if [ -n "$ENVOY_DOCKER_BUILD_DIR" ]; then
-  rm -rf "$ENVOY_DOCKER_BUILD_DIR/envoy/x64/bin/"
-else
-  rm -rf "/build/envoy/x64/bin/"
+if [ -z "$BUILD_DIR" ]; then
```

Comment:
You had mentioned something about this in a comment:

> we aren't patching envoy for env vars into do_ci.sh -- we need to set BUILD_DIR because if unset, everything else is overridden in build_setup.sh

Can you say more about this? I think I see [the block you're talking about: ](https://github.com/envoyproxy/envoy/blob/release/v1.29/ci/build_setup.sh#L83-L86)
```sh
if [[ -z "${BUILD_DIR}" ]]; then
    echo "BUILD_DIR not set - defaulting to ~/.cache/envoy-bazel" >&2
    BUILD_DIR="${HOME}/.cache/envoy-bazel"
fi
```

But does it matter whether we build in the default location?